### ISSUE TITLE
Many members, not enough check configs error state

### ISSUE THREAD

**Issue by dselans** [9corp/9volt#80]
Fix error condition:

If there are more members than available checks, the members will be fetching 'config' from members that don't actually have it (because they do not have any checks assigned to them). This is not critical, but it's a bit alarming (and it fills the event log as well).

Example:

```
ERRO[1920] director-collectCheckStats: Unable to fetch check stats: Unable to fetch memberRefs for FetchCheckStats(): Problem fetching refs for '52cbe95f': 100: Key not found (/9volt/cluster/members/52cbe95f/config) [1420697]
```

**Comment by dselans** [comment]
Update: If you have a large, *fresh* cluster with no checks; this causes *a lot* of event log spam. I'll take this up now.

**Comment by dselans** [comment]
This was resolved in #143
